src/main/java/org/computerspecsviewer/cli/Runner.java

```diff
@@ -1,7 +1,78 @@
 package org.computerspecsviewer.cli;
 
+import org.computerspecsviewer.infoquery.controller.InfoQueryController;
+import org.computerspecsviewer.infoquery.utils.PrintHelpers;
+import org.computerspecsviewer.infoquery.utils.StringHelpers;
+
+import java.util.Scanner;
+
 public class Runner {
     public static void main(String[] args) {
+        InfoQueryController iqc = new InfoQueryController();
+
+        Scanner inputReader = new Scanner(System.in);
+        setUpShutdownHook(inputReader);
+
+        while(true) {
+            printStartingInstructions();
+            String queryRequest = inputReader.next();
+
+            if(foundQuitInput(queryRequest)) {
+                break;
+            }
+
+            if(iqc.foundInfoType(queryRequest)) {
+                printInfoQuery(iqc, queryRequest);
+            } else if(queryRequest.equals("all")) {
+                for(String infoType: InfoQueryController.allInfoTypes) {
+                    printInfoQuery(iqc, infoType);
+                }
+            } else {
+                printInfoTypeNotFound(queryRequest);
+            }
+        }
+    }
+
+    private static void cleanUpRunner(Scanner scanner) {
+        printClosingStatements();
+        scanner.close();
+    }
+
+    private static void setUpShutdownHook(Scanner scanner) {
+        Thread closingHook = new Thread(() -> cleanUpRunner(scanner));
+        Runtime.getRuntime().addShutdownHook(closingHook);
+    }
+
+    private static void printInfoQuery(InfoQueryController iqc, String queryRequest) {
+        System.out.println();
+        System.out.println(StringHelpers.transformFieldName(queryRequest) + " Info:");
+        System.out.println(PrintHelpers.injectTabs(iqc.getInfoQuery(queryRequest).toString(), 1));
+    }
+
+    private static void printStartingInstructions() {
+        System.out.println("\n- Type one of these choices to view their information: [computerSystem, cpu, disks, " +
```

Comment:
Use an actual list instead of a hardcoded one, since this one matches with the one in InfoQueryController?